app/src/main/java/com/aileyzhang/musicapp/adapters/MainActivityContentAdapter.java

```diff
@@ -7,26 +7,36 @@
 import android.view.View;
 import android.view.ViewGroup;
 
+import com.aileyzhang.musicapp.fragments.AlbumListFragment;
 import com.aileyzhang.musicapp.fragments.SongListFragment;
 
 /**
  * Created by Ailey on 2017-12-12.
  */
 
 public class MainActivityContentAdapter extends FragmentPagerAdapter {
+    private final int TOTAL_PAGES = 2;
 
     public MainActivityContentAdapter(FragmentManager fm) {
         super(fm);
     }
 
     @Override
     public Fragment getItem(int position) {
-        return new SongListFragment();
+        Fragment fragment = null;
+        switch (position) {
+            case 0:
+                fragment = new SongListFragment();
+                break;
+            case 1:
+                fragment = new AlbumListFragment();
+        }
+        return fragment;
```

Comment:
Kinda a preference thing, but I would structure this by returning the fragment immediately in each case, then having a default case return `null` instead of using a local variable.